jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -91,18 +91,43 @@ spec:
             configMapKeyRef:
               name: jupyterhub-cfg
               key: notebook_destination
-        image: quay.io/odh-jupyterhub/jupyterhub-img:v0.3.1
+        - name: TRAEFIK_API_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: $(traefik_credentials_secret)
+              key: TRAEFIK_API_USERNAME
+        - name: TRAEFIK_API_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: $(traefik_credentials_secret)
+              key: TRAEFIK_API_PASSWORD
+        image: quay.io/odh-jupyterhub/jupyterhub-img:pr-104
         imagePullPolicy: "Always"
         name: jupyterhub
         ports:
         - containerPort: 8080
           protocol: TCP
         resources:
           limits:
+            cpu: "1"
+            memory: 1Gi
+          requests:
```

Comment:
Does it need to allocate a full cpu by default? Especially with the proxy being external now?